Tests/iaas/scs-0104-v1-images.yaml

```diff
@@ -0,0 +1,15 @@
+images:
+- name: "Ubuntu 22.04"
+  source: https://cloud-images.ubuntu.com/releases/jammy/
+  status: mandatory
+- name: "Kubernetes CAPI 1.27.3"
```

Comment:
k8s-cluster-api-provider uses and expects a different name here, see:
https://github.com/SovereignCloudStack/k8s-cluster-api-provider/blob/6d08e840273dcdb5f02edcc73f36b3d509822826/terraform/files/bin/upload_capi_image.sh#L24
`ubuntu2204-capi-image-v1.27.3`
This has been like this for a while.
So we either adjust the k8s-capi scripts to make the capi img useful or we adjust the standard to what the scripts use and expect.

Note 1: The canonical name according to the naming recommendations would probably be "Ubuntu 22.04 CAPI K8S v1.27.3" or so. That would require changes on both ends (standards and tooling).
Note 2: These CAPI images will change every month, as we have one patch level per month roughly in k8s ... So we would either have to accept recommending outdated things in the standard or to change the standard every month or so ...

Reply:
Proposal from the IaaS team call: Don't fix the k8s version as recommendation. Just define the naming scheme for the capi images.

Reply:
> the script above uses `ubuntu-22.04-capi-image-v1.27.3`

The referenced line would resolve to `ubuntu-2204-kube-v1.27.3` which is the (filename minus the .qcow2) that is used to download the node image from osism's minio.
The OpenStack image name used is defined by the `OPENSTACK_IMAGE_NAME` as defined in clusterctl.yaml.
We should just align our defaults here ... If we use names with a space, we'll need to do some extra testing to ensure the quoting is done correctly in all scripts.

Reply:
@garloff I opted for that (`ubuntu-22.04-capi-image-v1.27.3`) in the latest version, and if that goes into effect, then the CSPs have to align, and I guess they will (@berendt for instance told me that he doesn't have a strong opinion on that).

Reply:
And the default `OPENSTACK_IMAGE_NAME` in k8s-cluster-api-provider is defined here:
https://github.com/SovereignCloudStack/k8s-cluster-api-provider/blob/65af7217a4c2d849d1a408d36a3c9c6ea6c554aa/terraform/files/template/clusterctl.yaml.tmpl#L15
`OPENSTACK_IMAGE_NAME: ubuntu-capi-image-${kubernetes_version}`
So `ubuntu-capi-image-v1.XX.Y` not just in gx-scs.
My apologies for the confusion!

Going forward, I would like to ensure that:
* The default image-manager settings
* The recommended naming scheme in the std image spec
* The code in k8s-cluster-api-provider
all use the same. 

Right now my preference would be to stick to what we use in k8s-capi: `ubuntu-capi-image-v1.xx.yy`
If we change it, probably we should go to the canonical name. Doing this now, right before the release, is not optimal though.

Reply:
@garloff So the two code occurrences contradict each other? This should be fixed, no? I'm fine with using either one of the two, and I would not go through the hassle of changing it to a third option. Still, the code occurrences should be harmonized, shouldn't they?

Reply:
There is no contradiction, just my own confusion, for which I apologise.
To create a cluster with Kubernetes v1.xx.yy, the current k8s-cluster-api-provider scripts look for an image ubuntu-capi-image-v1.XX.YY. (This has been like this since R2 or so.)
If the image does not exist, a file ubuntu-2204-kube-v1.XX.YY.qcow2 would be downloaded from osism minio and registered as ubuntu-capi-image-v1.XX.YY image. (This has been like this as well for a long time, except that the -2204 was added in when we switched to 22.04.)

So, ubuntu-capi-image-v1.XX.YY it is from a k8s-cluster-api-provider perspective.

Reply:
_If_ the image is downloaded if it does not exist (which is a very good fallback, since it is self contained), I see no reason to actually reference it in the Iaas-related standard. Or do I overlook something?

Reply:
If the image already has been registered before, cluster creation will be a few minutes. Download and registration typically add 10mins to that. In addition you get a private image, which is inefficient if many people / projects use capi. So a recommendation to the IaaS provider would seem useful to me. Better user experience, less wasted storage.